docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,22 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, str]:
+
+    if tags.has('redirects'):
+
+        redirects = {}
+        redirects_config_path = DOCS_ROOT_DIR.parent / "ansible_redirects.toml"
+        with open(redirects_config_path, "rb") as f:
+            redirects = tomllib.load(f)
```

Comment:
@webknjaz I suppose there might be other ways to parse that toml file but this was suggested in the docs: https://docs.python.org/3/library/tomllib.html#examples